.github/workflows/upgrade-tests.yml

```diff
@@ -66,6 +66,17 @@ jobs:
           # if empty, defaults to all test kinds within script
           TEST_KINDS="${{ github.event.inputs.test_kinds }}"
 
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            # the GH runners become flaky with manual runs during the release
+            # process, so we reduce parallelism for manual release testing
+            FM_TEST_CI_ALL_JOBS="5"
+            # some upgrade paths can take a long time, so we increase the timeout
+            FM_TEST_UPGRADE_TIMEOUT="3600"
+          else
+            FM_TEST_CI_ALL_JOBS=""
```

Comment:
Nit: bash has `unset` which is generally better than setting env vars to `""` and hoping that nothing interprets `""` as "set".

Reply:
@bradleystachurski just making sure you've seen it as it was already in the mq

Reply:
TIL

Reply:
We've figured it out, will get landed in the near future.

Reply:
https://github.com/fedimint/fedimint/pull/6750